### PR TITLE
Adds types for bloomfilter

### PR DESCRIPTION
**Pull request by slawiko**
PR for issue #15183 

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `tsc` without errors.
- [ ] Run `npm run lint package-name` if a `tslint.json` is present.

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `npm run new-package package-name`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

File: bloomfilter/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// Type definitions for BloomFilter v0.0.16
+// Project: https://github.com/jasondavies/bloomfilter.js
+// Definitions by: slawiko https://github.com/slawiko
```
Comment (paulvanbrenk) [inline bloomfilter/index.d.ts]: change this to 
`// Definitions by: slawiko <https://github.com/slawiko>`

**Comment by paulvanbrenk** [comment]
thanks for the contribution, can you make sure the CI server passes.

**Comment by slawiko** [comment]
@paulvanbrenk can you help me, please? As I understand Travis says that `buckets: Array<Int32Array>;` is wrong, because of brackets `<>`?

File: bloomfilter/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (paulvanbrenk) [inline bloomfilter/tsconfig.json]: `tsconfig.json` is wrong, should be similar to:

```json
{ 
    "compilerOptions": { 
        "module": "commonjs", 
        "lib": [ 
            "es6", 
            "dom" 
        ], 
        "noImplicitAny": true, 
        "noImplicitThis": true, 
        "strictNullChecks": true, 
        "baseUrl": "../", 
        "typeRoots": [ 
            "../" 
        ], 
        "types": [], 
        "noEmit": true, 
        "forceConsistentCasingInFileNames": true 
    }, 
    "files": [ 
        "index.d.ts", 
        "bloomfilter-tests.ts"
        ] 
}
```

File: bloomfilter/index.d.ts (hunk 1 above)
Comment (slawiko) [inline bloomfilter/index.d.ts]: Done

File: bloomfilter/tsconfig.json (hunk 2 above)
Comment (slawiko) [inline bloomfilter/tsconfig.json]: Done

**Comment by slawiko** [comment]
@paulvanbrenk [this](http://definitelytyped.org/guides/contributing.html) article has this line:

> If the version of the library is known then add it as a semver to the label.\

But if I add bloomfilter version in `index.d.ts` like I did it [here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/15256/files/45f48bde141128e0554b1fa78f0ae1ba035700d6#diff-dd75e1eb7274e34d3ad0dda083eeaa31) Travis tells me `Expected: foo MAJOR.MINOR ('v' not allowed)`. It's my fault or this article has faults?

**Comment by paulvanbrenk** [comment]
@slawiko thanks, it looks like we tightened the check for the version a little too much.

**Pull request merged**